### PR TITLE
docs: changelog for 05-31-2024

### PR DESCRIPTION
**Pull request by danieltprice**
https://neon-next-git-docs-changelog-05-31-2024-neondatabase.vercel.app/docs/changelog

Items tagged as [05/31] are not on prod yet. If they are not there by tomorrow afternoon, they will be stashed for next week.

### CODE REVIEW

**Review by ruf-io — APPROVED** [review]

File: content/changelog/2024-05-31.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+The benefits of joining include:
+
+- **Exclusive early access**: Get a first look at upcoming features before they go live.
+- **Private community**: Gain access to a dedicated Discord channel to connect with the Neon team and provide feedback to help shape what comes next.
+- **Weekly insights**: Receive updates on Neon's latest developments and future plans.
+
+### Computes, roles, and databases moved to branch pages in the Neon Console
+
+Computes, roles, and databases are now accessed from individual branch pages in the Neon Console instead of the Neon Console sidebar. Each of those objects is associated with a particular branch in a Neon project, and this change is intended to reflect that relationship.
```
Comment (bgrenon) [inline content/changelog/2024-05-31.md]:
```suggestion
You can now find your branch's computes, roles, and databases on the branch page they belong to. This update better reflects the relationship of these objects to their specific branches in a Neon project.
```

File: content/changelog/2024-05-31.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+![new branches page](/docs/relnotes/new_branch_page_with_tabs.png)
+
+With this change, the compute endpoint delete option was moved to the **Edit compute endpoint** drawer, accessed by clicking **Edit** on the **Computes** tab.
```
Comment (bgrenon) [inline content/changelog/2024-05-31.md]:
```suggestion
With this change, we also moved the compute endpoint delete option to the **Edit compute endpoint** drawer, which you can access by clicking **Edit** on the **Computes** tab.
```

**Review by bgrenon — APPROVED** [review]
Looks good!

File: content/changelog/2024-05-31.md (hunk 2 above)
Comment (danieltprice) [reply content/changelog/2024-05-31.md]: I removed this content, so I'll apply these suggestions to our stashed items for next week! Thank you!

**Pull request merged**